### DISCUSSION TITLE
Cannot start IotDB

### DISCUSSION THREAD

**Discussion by A-M-K-G** [apache/iotdb#3950] [Q&A] [answered]
IotDB运行一段时间之后自动关闭了，尝试用sh start-server.sh 命令重新启动时出现了如下信息：
![1](https://user-images.githubusercontent.com/89012709/133015868-fdd51037-db62-47d5-b15b-653906ba49c0.png)
请问这种情况应该如何解决呢

**Comment by HTHou** [comment]
可以检查一下机器内存有多大，会不会是因为内存不足导致 JVM 被操作系统 kill 掉了

**Reply by qiaojialin** [reply]
用后台启动试试

  nohup ./start-server.sh >/dev/null 2>&1 &

**Reply by A-M-K-G** [reply]
内存应该还是比较充足的
![2](https://user-images.githubusercontent.com/89012709/133019246-3c6118e5-a937-422e-8b28-76373d248231.png)
 后台启动命令也尝试过了，也是不能够启动

**Comment by HeimingZ** [comment, marked as answer]
请问是哪个版本。重启前进行了哪些操作（越详细越好），有没有重启前的日志呢。

**Reply by A-M-K-G** [reply]
问题已经解决，是服务器被病毒入侵导致所有其他进程被关闭，不是IotDB的问题，谢谢各位的帮助
